File: src/main/java/com/github/aborn/webdevtoolkit/services/MyProjectService.java
package com.github.aborn.webdevtoolkit.services;

import com.intellij.openapi.components.Service;
import com.intellij.openapi.project.Project;

/**
 * @author aborn
 * @date 2021/01/29 2:44 PM
 */
@Service
public final class MyProjectService {
    private final Project myProject;

    public MyProjectService(Project project) {
        myProject = project;
    }
}

File: src/main/kotlin/com/github/aborn/webdevtoolkit/services/MyProjectService.kt
package com.github.aborn.webdevtoolkit.services

import com.github.aborn.webdevtoolkit.MyBundle
import com.intellij.openapi.project.Project

class MyProjectService(project: Project) {

    init {
        println(MyBundle.message("projectService", project.name))
    }
}
